.clang-format

```diff
@@ -11,6 +11,7 @@ AllowAllParametersOfDeclarationOnNextLine: true
 AllowShortBlocksOnASingleLine: Empty
 AllowShortCaseLabelsOnASingleLine: false
 AllowShortFunctionsOnASingleLine: Empty
+AllowShortLambdasOnASingleLine: Empty
```

Comment:
What does empty mean here? Not an empty lambda, or does it?

Reply:
Oh, seems like it does. Hm, guess that's okay. Do we already have that in the guidelines? Not really sure if that is always better, e.g., if
```c++
std::sort(vec.begin(), vec.end(), [](const auto& lhs, const auto& rhs) {
  return lhs < rhs;
});
```

is more readable than

```c++
std::sort(vec.begin(), vec.end(), [](const auto& lhs, const auto& rhs) { return lhs < rhs; });
```

but I get the rationale.

Reply:
I wasn't sure as well in the first place. But I think for most code path, I thinks the code is more readable. As it is better align with the rest of the code.

There are a few cases where it looks pretty ridicules, BUT that's mostly the case when we nest multiple lambdas. And in some of the cases I think we went too far with nesting.

So in general, I would go with this change. If you see an problems with it, we would also consider clang-format exceptions as we do for the LQP  query constructions in the tests.